conda_smithy/configure_feedstock.py

```diff
@@ -1707,6 +1707,10 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         "drone": {},
         "woodpecker": {},
         "travis": {},
+        "travis_linux_aarch64_instance": {
+            "name" : "arm64",
+            "size" : "medium",
+        },
```

Comment:
I don't think we need a separate key. We can do just like azure.

Reply:
Thanks, updated the PR based on your feedback.